lib/ae_test_coverage/collectors/webpacker/helpers.rb

```diff
@@ -4,12 +4,12 @@ module AeTestCoverage
   module Collectors
     module Webpacker
       module Helpers
-        def javascript_pack_tag(*names, **options)
+        def append_javascript_pack_tag(*names, **options)
           add_to_coverage(names)
           super
         end
 
-        def javascript_packs_with_chunks_tag(*names, **options)
+        def javascript_pack_tag(*names, **options)
```

Comment:
is there a reason we want to get rid of the old helpers? even if they aren't called at the moment for safety reasons is not a good idea to keep them around in case someone uses them? or is it purely tech debt?

Reply:
Well shakapacker doesn't support these anymore, so those tests will fail. I can keep them and remove the tests correlated to those

Reply:
sounds good - sounds like they can go then